app/src/main/java/org/fossasia/susi/ai/activities/LoginActivity.java

```diff
@@ -99,8 +99,25 @@ public void onResponse(Call<LoginResponse> call, Response<LoginResponse> respons
                     intent.setFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP | Intent.FLAG_ACTIVITY_NEW_TASK);
                     startActivity(intent);
                     finish();
-                } else
-                    Toast.makeText(LoginActivity.this, response.message(), Toast.LENGTH_SHORT).show();
+                } else {
+
+                    AlertDialog.Builder builder = new AlertDialog.Builder(LoginActivity.this);
+                    builder.setMessage(R.string.password_invalid)
+                            .setCancelable(false)
+                            .setPositiveButton("OK", new DialogInterface.OnClickListener() {
```

Comment:
.setPositiveButton("OK", null) 
could reduce the code :)
It would be great if you can add a title also like "Incorrect Password"